examples/build-dependencies/internal-model/build.gradle.kts

```diff
@@ -0,0 +1,15 @@
+// This example builds the model and places it in the JAR.
+
+plugins {
+    `java-library`
+    id("software.amazon.smithy").version("0.7.0")
+}
+
+repositories {
+    mavenLocal()
+    // mavenCentral()
```

Comment:
Is this intended to be commented out?

Reply:
Yes. I will add a comment to this effect.